pkg/rules/queryable.go

```diff
@@ -81,7 +81,7 @@ func (q *promClientsQueryable) Querier(ctx context.Context, mint, maxt int64) (s
 }
 
 // Select implements storage.Querier interface.
-func (q *promClientsQuerier) Select(_ bool, _ *storage.SelectHints, matchers ...*labels.Matcher) storage.SeriesSet {
+func (q *promClientsQuerier) Select(ctx context.Context, _ bool, _ *storage.SelectHints, matchers ...*labels.Matcher) storage.SeriesSet {
```

Comment:
Since we have `ctx` here, we should change L91 to use this ctx

Reply:
:heavy_check_mark: